decocare/history.py

```diff
@@ -224,12 +224,42 @@ class Ian54(KnownRecord):
 _confirmed.append(Ian54)
 
 class SensorAlert (KnownRecord):
+  """Glucose sensor alarms.
+    The second byte of the head represents the alarm subtype.
+    The third byte seems to contain an alarm-specific value.
+
+    For example, a "Low Glucose" alarm is:
+    [
+      0x0b,  # 11: Opcode
+      0x66,  # 102: Low glucose subtype
+      0x50   # 80: Glucose level (For a pump configured to mg/dL)
+    ]
+    """
   opcode = 0x0B
   head_length = 3
-  def decode (self):
-    self.parse_time( )
-    amount = self.head[2]
-    return dict(alarm_type=self.head[1], amount_maybe=amount)
+
+  alarm_subtypes = {
```

Comment:
These are descriptive, which probably isn't the best for clients to handle. I'm happy to keep the original type code as `'alarm_type'` and include the text as `'alarm_description'`, or, if you had a cross-platform enum in mind for openaps we could use that instead